docs/WritingFirstTest-README.md

```diff
@@ -0,0 +1,97 @@
+# Writing the first tests
+
+## Pre-requisites:
+
+        1. Have admin rights  
+        2. Intellij (install cucumber JVM plugin)
+        3. Java JDK 11  
+        4. Gradle 7.1
```

Comment:
Gradle 7.3.3
Also, we do not want people to have gradle installed. We use gradle wrapper